### PR TITLE
Add task to remove duplicated table vizs

### PR DESCRIPTION
**Pull request by amartinj**
Closes https://github.com/CartoDB/cartodb-central/issues/2205

### CODE REVIEW

File: spec/lib/tasks/viz_mainteinance_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    after(:all) do
+      ::User[@user.id].destroy
+    end
+
+    it 'deletes duplicated table visualizations' do
+      map, table, table_visualization, visualization = create_full_visualization(@user)
+      table_visualization.updated_at = @table_visualization.updated_at + 1.minute
+      table_visualization.save(validate: false)
+      table_visualization.update_column(:name, @table_visualization.name)
```
Comment (javitonino) [inline spec/lib/tasks/viz_mainteinance_spec.rb]: You also need to update the name of the user table (visualization.user_table) to replicate the situation in production.

File: lib/tasks/viz_maintenance.rake [hunk 2]
```diff
@@ -212,6 +212,32 @@ namespace :cartodb do
...
+                v.user_id,
+                v.name,
+                v.type
+              having
+                v.type = 'table' and
+              count(*) > 1;"
+      ActiveRecord::Base.connection.execute(query).each do |row|
+        begin
+          vizs = Carto::User.find(row['user_id']).visualizations.where(name: row['name']).all
+          vizs.sort_by(&:updated_at).slice(0..vizs.size - 2).each(&:destroy)
```
Comment (javitonino) [inline lib/tasks/viz_maintenance.rake]: :warning: Destroying the visualization will destroy the actual data table.

File: spec/lib/tasks/viz_mainteinance_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+      Carto::Visualization.where(type: 'table').count.should eq 3
+      Rake.application['cartodb:vizs:remove_dup_table_vizs'].invoke
+
+      Carto::Visualization.where(type: 'table').count.should eq 2
+      Carto::Visualization.find(table_visualization.id)
+      Carto::Visualization.find(@table_visualization2)
+      expect { Carto::Visualization.find(@table_visualization.id) }.to raise_error(ActiveRecord::RecordNotFound)
```
Comment (javitonino) [inline spec/lib/tasks/viz_mainteinance_spec.rb]: Please also check that the actual database table still exists.

**Comment by amartinj** [comment]
Proper fix in https://github.com/CartoDB/cartodb/pull/13859

**Comment by amartinj** [comment]
Closes https://github.com/CartoDB/cartodb-central/issues/2205

**Pull request closed without merging**